erp-client/src/app/models/new-user.model.ts

```diff
@@ -0,0 +1,10 @@
+import {RecoveryQuestion} from './recovery.model';
+
+export class NewUser {
+  email: string;
+  username: string;
+  password: string;
+  rePassword: string;
+  recoveryQuestions: RecoveryQuestion[];
```

Comment:
Will I need to go back and add this for her creation by admins?

Reply:
I'm not sure what you mean

Reply:
Yea..I forgot we need to set recovery questions. Strange for the admin to put those and the persons signature in, but I haven't really thought of any other way to do the admin creating users.